src/index.html

```diff
@@ -57,6 +57,9 @@
              */
             outline: 1px solid transparent;
         }
+        #iframe-warning {
+            visibility: hidden;
+        }
```

Comment:
Let's add `padding: 20px;` to this too, so it isn't so tight to the edge of the window.